migrations/20240123170426_skills-db.ts

```diff
@@ -0,0 +1,25 @@
+import type { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('skills', function (table) {
+    table.increments('id');
+    table.string('skill');
+    table.string('grade_1');
```

Comment:
Думаю, лучше будет так
```suggestion
    table.string('grade_1_desc');
```